diff -urNp libkrisp.org/Makefile.in libkrisp/Makefile.in
--- libkrisp.org/Makefile.in	2011-04-26 17:58:48.000000000 +0900
+++ libkrisp/Makefile.in	2011-04-27 17:13:05.000000000 +0900
@@ -363,6 +363,8 @@ install-libLTLIBRARIES: $(lib_LTLIBRARIE
 	    f=$(am__strip_dir) \
 	    echo " $(LIBTOOL) --mode=install $(libLTLIBRARIES_INSTALL) $(INSTALL_STRIP_FLAG) '$$p' '$(DESTDIR)$(libdir)/$$f'"; \
 	    $(LIBTOOL) --mode=install $(libLTLIBRARIES_INSTALL) $(INSTALL_STRIP_FLAG) "$$p" "$(DESTDIR)$(libdir)/$$f"; \
+	    echo "/usr/bin/chrpath -d '$(DESTDIR)$(libdir)/libkrisp.so.$(SONAME_VERSION)'"; \
+		/usr/bin/chrpath -d "$(DESTDIR)$(libdir)/libkrisp.so.$(SONAME_VERSION)"; \
 	  else :; fi; \
 	done
 
diff -urNp libkrisp.org/utils/Makefile.in libkrisp/utils/Makefile.in
--- libkrisp.org/utils/Makefile.in	2011-04-26 17:58:48.000000000 +0900
+++ libkrisp/utils/Makefile.in	2011-04-27 17:10:14.000000000 +0900
@@ -256,6 +256,8 @@ install-binPROGRAMS: $(bin_PROGRAMS)
 	    f=`echo "$$p1" | sed 's,^.*/,,;$(transform);s/$$/$(EXEEXT)/'`; \
 	   echo " $(INSTALL_PROGRAM_ENV) $(LIBTOOL) --mode=install $(binPROGRAMS_INSTALL) '$$p' '$(DESTDIR)$(bindir)/$$f'"; \
 	   $(INSTALL_PROGRAM_ENV) $(LIBTOOL) --mode=install $(binPROGRAMS_INSTALL) "$$p" "$(DESTDIR)$(bindir)/$$f" || exit 1; \
+	   echo "/usr/bin/chrpath -d '$(DESTDIR)$(bindir)/$$f'"; \
+	   /usr/bin/chrpath -d "$(DESTDIR)$(bindir)/$$f"; \
 	  else :; fi; \
 	done
 
